tests/system_banner/test_system_banner_message.py

```diff
@@ -0,0 +1,155 @@
+import allure
+import logging
+
+logger = logging.getLogger()
+
+CONFIGURE_BANNER_CMD = "config banner "
+SHOW_BANNER_CMD = "show banner"
+NO_BANNER = "no banner "
+PASSWORD = "admin"
+FAILED = "failed"
+DEF_STATE = "disabled"
+NEW_STATE = "enabled"
+DEF_LOGIN_MSG = "Debian GNU/Linux 11"
+DEF_MOTD_MSG = "This is temporary default MOTD"
+DEF_LOGOUT_MSG = ""
+NEW_LOGIN_MSG = "Welcome to SONIC CLI"
+NEW_MOTD_MSG = "This is SONIC"
+NEW_LOGOUT_MSG = "Good Bye!"
+SUDO_STR = "sudo "
```

Comment:
do we have these consts in a more general place so we can reuse them>

Reply:
There is no such common place in any library. U have verified that the other tests are having their constants in the same file